scripts/setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 confirm_api_url () {
-    echo "Please input your API_URL. e.g. https://api-media.bucketeer.jp"
+    echo "Please input your API_URL. e.g. https://api.bucketeer.jp"
```

Comment:
Please change this to `https://api.bucketeer.io`

Reply:
Fixed https://github.com/ca-dp/bucketeer-ios-sdk/pull/92/commits/24db46aa5f0e4e6945ac3521a0277ef2d472f489